types/react-twemoji/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for react-twemoji 0.4
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
please review this line, update url and remove template comment.
The project seems to have a home page:
https://github.com/zxmys/react-twemoji